cppwinrt/main.cpp

```diff
@@ -374,5 +375,7 @@ Where <spec> is one or more of:
 
 int main(int const argc, char** argv)
 {
+    // Dynamically enable long path support
+    ((unsigned char*)(NtCurrentTeb()->ProcessEnvironmentBlock))[3] |= 0x80;
```

Comment:
We should not be calling undocumented APIs from cppwinrt.exe.

Reply:
Is there an alternative or do we pull it out?

Reply:
An alternative is to create an app manifest; I had tried doing that, the manifest was getting embedded, but long path were still not working. We could pull out this one line from the code and leave the error reporting, and I can try to rework the manifest. LMK what you'd like to do.

Reply:
PR to use the manifest here: https://github.com/microsoft/cppwinrt/pull/882